packages/backend-defaults/src/entrypoints/rootHttpRouter/rootHttpRouterServiceFactory.ts

```diff
@@ -85,6 +85,13 @@ const rootHttpRouterServiceFactoryWithOptions = (
       const logger = rootLogger.child({ service: 'rootHttpRouter' });
       const app = express();
 
+      const trustProxy = config.getOptional<boolean | number>(
+        'backend.trustProxy',
+      );
+      if (trustProxy) {
+        app.set('trust proxy', trustProxy);
+      }
```

Comment:
Hm does this belong here or inside `applyDefaults()`? I guess both work, if you can overwrite this value inside your custom `configure` just fine. But having it inside `applyDefaults` does align with doing all "actual work" in there.